packages/wrangler/src/config/index.ts

```diff
@@ -24,29 +25,32 @@ export type {
 	ConfigModuleRuleType,
 } from "./environment";
 
-type ReadConfigCommandArgs<CommandArgs> = CommandArgs &
-	Pick<OnlyCamelCase<CommonYargsOptions>, "experimentalJsonConfig"> &
-	Partial<Pick<OnlyCamelCase<CommonYargsOptions>, "env">>;
+type ReadConfigCommandArgs = Pick<
```

Comment:
`env` is included in this type twice, I think?

Reply:
Transiently, yes, but I don't think that's a problem.

To be honest, I'd rather spell this type out literally than doing these `Pick`'s